R/scenario_sim.R

```diff
@@ -35,14 +35,14 @@
 #' )
 #' res
 scenario_sim <- function(n, prop_ascertain, cap_max_days, cap_cases,
-                         r0isolated, r0community, disp_isolated, disp_community, k,
+                         r0_isolated, r0_community, disp_isolated, disp_community, k,
                          onset_to_isolation, incubation_period,
                          initial_cases, prop_asymptomatic, quarantine = FALSE,
-                         r0asymptomatic = NULL, disp_asymptomatic = NULL) {
+                         r0_asymptomatic = NULL, disp_asymptomatic = NULL) {
```

Comment:
same a previous comment - probably want isolated vs community vs asymptomatic grouped together in arguments.

Reply:
I'll address this in a separate PR with #93.